src/NzbDrone.Core/Indexers/IndexerStatusService.cs

```diff
@@ -54,12 +54,15 @@ public void UpdateRssSyncStatus(int indexerId, ReleaseInfo releaseInfo)
 
         public void UpdateCookies(int indexerId, IDictionary<string, string> cookies, DateTime? expiration)
         {
-            lock (_syncRoot)
+            if (indexerId > 0)
```

Comment:
This will prevent indexers with 0 from being added to table during test call before add